src/library.js

```diff
@@ -3272,8 +3272,10 @@ addToLibrary({
 #if !EXIT_RUNTIME && ASSERTIONS
     warnOnce('emscripten_force_exit cannot actually shut down the runtime, as the build does not have EXIT_RUNTIME set');
 #endif
-#if !MINIMAL_RUNTIME
+#if '$noExitRuntime' in addedLibraryItems
```

Comment:
Does it work to check `addedLibraryItems` in `library.js`? It does work in `jsifier.js` and `preamble.js` for example but only because the library items are calculated first. But aren't the library items still being calculated as we parse `library.js`?

Reply:
Hmm.. maybe you are right.    Its a little concerning that the tests didn't catch that.